reward_index/migrations/5_add_type_to_index.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TYPE reward_type as enum('mobile', 'iot_gateway', 'iot_operational');
```

Comment:
no `'mobile_operations'`? i don't see an equivalent operational funding percentage in the hip53 breakdown; is that accurate?

Reply:
yes there is no `mobile_operations`